src/dune_rules/dir_contents.mli

```diff
@@ -50,13 +54,11 @@ type gen_rules_result =
     rules.
 
     As a result, we proceed as follow: we interpret the stanza into rules and
-    extract the targets of the computed rule. This function simply emits these
-    rules so that they can be collected by [Build_system].
+    extract the targets of the computed rule. This function returns these rules.
 
     However, if the directory is part of a group, this function simply returns
     the root of the group without emitting any rule. *)
```

Comment:
This function no longer "emits" rules, so this sentence should be adjusted.

Reply:
Indeed, changed it.